site/docs/components/overlay/index.mdx

```diff
@@ -0,0 +1,27 @@
+---
+title: Overlay
+data:
+  description: "`Overlay`  reveals supplementary content when the user clicks a UI trigger element. It remains active and open until the user dismisses it. It can contain interactive and focusable elements, such as buttons and links, in addition to text elements."
+
+  # Fill in the info from the content template's "Metadata" table below.
+  # To omit optional items, comment them out with #
+  sourceCodeUrl: "https://github.com/jpmorganchase/salt-ds/tree/main/packages/lab/src/overlay"
+  package:
+    name: "@salt-ds/lab"
+  alsoKnownAs: ["Dialog", "Modal"]
```

Comment:
Where did we get these from?

Reply:
I'm a bit conflicted about it, we already have a Dialog component which is also known as a Modal. Having two with the same aka seems confusing. I also wouldn't expect someone who is looking for a Dialog to want an overlay

Reply:
that was included as part of my initial draft for Overlay docs. I think Popover works better than the above 2, @joshwooding what others do you reckon would be best suited for aka here? 

cc @bhoppers2008 @mikearildbrown please feel free to amend/update accordingly